### PR TITLE
Feature/examples section

### PR DESCRIPTION
**Pull request by pthoang**
## Description
Added some examples that make us of the components and features provided by Design Guide

### CODE REVIEW

File: src/App/Examples/web/customers/constants.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    {
+        id: "c7",
+        firstName: "Helma",
+        lastName: "Egilsdóttir",
+        email: "helma.egilsdottir@email.com",
+        phone: "+47 *** ** ***",
+        location: "Reykjavík",
+        status: "default"
+    }
+];
```
Comment (VikingTristan) [inline src/App/Examples/web/customers/constants.js]: faker.js

Comment (VikingTristan) [reply src/App/Examples/web/customers/constants.js]: Actually, let's not. :)

File: src/App/Examples/web/customers/index.js [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                        </header>
+                        <div className="panel-table">
+                            <table className="table table-hover table-striped">
+                                <thead>
+                                    <tr>
+                                        <th scope="col">ID</th>
+                                        <th scope="col" className="d-none d-sm-table-cell">First name</th>
```
Comment (hmosvoll) [inline src/App/Examples/web/customers/index.js]: What is the reason for `d-none d-sm-table-cell`?

File: src/App/Examples/web/customers/index.js [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+    CustomersOverview,
+    CustomersDetailed,
+    CustomersDetailedDatePickerGroup,
+    CustomersDetailedOrders,
+    CustomersDetailedInquiries,
+    CustomersDetailedInquiryCard,
+    CustomersDetailedSettings,
+    CustomersDetailedCharts,
+    CustomersDetailedDialog
+};
```
Comment (hmosvoll) [inline src/App/Examples/web/customers/index.js]: This file is quite big? Abstract into smaller files?

File: src/App/Examples/getting-started/Introduction/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { DocContainer } from "@docutils";
+
+const brandTitle = process.env.brandTitle;
+
+const Introduction = () => (
+    <DocContainer docToc>
+        <p className="lead">Getting started with the {brandTitle} examples. Examples and inspiration on how to use parts provided by the Design Guide can be found here.</p>
```
[the remark this answers is not in the document]
Comment (pthoang) [reply src/App/Examples/getting-started/Introduction/index.js]: Probably not. Just something I left there when copying over from the docs code. Will remove be removed

File: src/App/Examples/web/customers/index.js (hunk 2 above)
Comment (pthoang) [reply src/App/Examples/web/customers/index.js]: This is just me playing with ways to present the table on smaller devices. Some of the columns are hidden when the screen is too narrow to show all of the content

File: src/App/Examples/web/customers/index.js (hunk 3 above)
Comment (pthoang) [reply src/App/Examples/web/customers/index.js]: Good idea :)

File: src/App/Examples/web/faq/index.js [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+                        <div className="media-img">
+                            <i className="material-icons text-muted">help</i>
+                        </div>
+                        <div className="media-body">
+                            <h4 className="text-muted">Cannot find what you are looking for?</h4>
+                            <p><small>Or any other inqueries?</small></p>
+                        </div>
+                    </div>
+                    <p>Send in a question to get help from our support team</p>
+                    <button type="button" className="btn btn-executive btn-block mt-auto" data-sheet-open="faq-sheet">
```
[the remark this answers is not in the document]
Comment (pthoang) [reply src/App/Examples/web/faq/index.js]: Probably something stuck after different implementations. Will be removed

File: src/App/Examples/web/customers/index.js (hunk 2 above)
Comment (hmosvoll) [reply src/App/Examples/web/customers/index.js]: aah, I see.

**Comment by hmosvoll** [comment]
Nice. 
A Couple of comments:
- The image on FAQ is still really large on big displays.
- What is the reasoning behind "Profile description" and "User status" on the same line.

**Pull request merged**